cristin-person/src/main/java/no/unit/nva/cristin/person/model/cristin/CristinPersonSummary.java

```diff
@@ -0,0 +1,71 @@
+package no.unit.nva.cristin.person.model.cristin;
+
+import static java.util.Objects.nonNull;
+import static no.unit.nva.cristin.model.Constants.BASE_PATH;
+import static no.unit.nva.cristin.model.Constants.DOMAIN_NAME;
+import static no.unit.nva.cristin.model.Constants.HTTPS;
+import static no.unit.nva.cristin.model.Constants.PERSON_PATH_NVA;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.net.URI;
+import no.unit.nva.cristin.person.model.nva.PersonSummary;
+import nva.commons.core.paths.UriWrapper;
+
+public record CristinPersonSummary(@JsonProperty(FIRST_NAME) String firstName,
```

Comment:
I really prefer this way of declaring a record, one line for each field, just like we declare a normal class.
It is much easier to read.